python/test/unit/language/test_core.py

```diff
@@ -2671,7 +2669,7 @@ def kernel(X, Z, BLOCK_M: tl.constexpr, BLOCK_N: tl.constexpr, BLOCK_K: tl.const
 
     # triton result
     z_shape = z_ref.shape
-    z_tri = to_triton(numpy_random(z_shape, dtype_str=z_dtype_str, rs=rs), device=device, dst_type=z_tri_dtype_str)
+    z_tri = to_triton(numpy_random(z_shape, dtype_str=z_dtype_str), device=device, dst_type=z_tri_dtype_str)
```

Comment:
If we don't pass `rs` won't we get the same result as the previous call?

Reply:
The rs here was meaningless...as the default rs is already `rs = RandomState(17)`

Reply:
But the state is updated each time you generate more numbers from it

Reply:
Oh, you're referring to this `z_tri` array specifically? IMO it also doesn't matter as it's used to store the resultant array anyway. We can even use `np.zeros`, but here we use `numpy_random` probably because some data types (e.g., bfloat16) need extra code